debian/copyright

```diff
@@ -1,30 +1,36 @@
-This is Grml's GnuPG keyrings of archive keys.
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 
-This package was originally put together by Michael Vogt
-<mvo@debian.org> for Debian and has been adapted to grml.org by
-Alexander Wirt <formorer@grml.org> in 2007. Minor updates were done by
-Michael Prokop <mika@grml.org> and Chris Hofstaedtler <ch@grml.org>
-in the years 2007 to 2025.
+Files:
+ *
+Copyright:
+ 1998-2003 James Troup <james@nocrew.org>
+ 2006 Michael Vogt <mvo@debian.org>
+ 2007 Alexander Wirt <formorer@grml.org>
+ 2007-2025 Michael Prokop <mika@grml.org>
```

Comment:
nothing from James, Michael Vogt nor Alexander is left behind AFAICS, please drop those and add yourself instead! :)

Reply:
Ack! :)